packages/cdk/resources/Dynamodb/dynamodbResources.ts

```diff
@@ -0,0 +1,63 @@
+import {Construct} from "constructs"
+import {ManagedPolicy, PolicyStatement} from "aws-cdk-lib/aws-iam"
+import {TableV2} from "aws-cdk-lib/aws-dynamodb"
+
+/**
+ * Common resources for a dynamodb table
+
+ */
+
+export interface DynamodbResourcesProps {
+  readonly stackName: string
+  readonly table: TableV2
+}
+
+export class DynamodbResources extends Construct {
```

Comment:
I'm not sure why we're separating these into their own file. Why not just declare them in packages/cdk/resources/Dynamodb.ts

Reply:
fixed